eos-image-boot-dm-setup

```diff
@@ -76,7 +76,7 @@ while read extent_i extent_size; do
 
   if [ ${i} -lt ${extent_i} ]; then
```

Comment:
On a quick re-read, you are right, this should always be true. I can't remember the original reason for having this check here.